lib/fake_ifdh.py

```diff
@@ -50,10 +50,22 @@ def getExp() -> Union[str, None]:
 
 def getRole(role_override: Optional[str] = None) -> str:
     """get current role"""
+
     if role_override:
         return role_override
-    if os.environ["USER"][-3:] == "pro":
-        return "Production"
+
+    # if there's a role in the wlcg.groups of the token, pick that
+    if os.environ.get("BEARER_TOKEN_FILE", False):
+        with os.popen("decode_token.sh $BEARER_TOKEN_FILE", "r") as f:
+            token_s = f.read()
+            token = json.loads(token_s)
+            groups: List[str] = token["wlcg.groups"]
```

Comment:
Can we either have this be a `token.get` call that will return `DEFAULT_ROLE` if it finds no `wlcg.groups` key, or a `try-except` block that does the same if it catches `KeyError`?  Just in case our token is malformed.